img2dataset/downloader.py

```diff
@@ -75,9 +75,14 @@ def __init__(self, image_size, resize_mode, resize_only_if_bigger):
 
     def __call__(self, img_stream):
         try:
-            img = cv2.imdecode(np.frombuffer(img_stream.read(), np.uint8), 1)
+            img = cv2.imdecode(np.frombuffer(img_stream.read(), np.uint8), cv2.IMREAD_UNCHANGED)
             if img is None:
                 raise Exception("Image decoding error")
+            if len(img.shape) == 3 and img.shape[-1] == 4:
```

Comment:
You may to want to make the alpha color configurable. Otherwise, this looks good. I also added a comment about adding proper rounding.

Reply:
What do you think @rom1504 ?
You want to be able to choose the background color (instead of white) or we wait to see if it's requested?

Reply:
no I think let's keep white only for now so it's coherent with the resize border color

we may add that option if anybody can think of any use, let's not add it now